internal/layer2/announcer.go

```diff
@@ -337,3 +350,10 @@ const (
 	dropReasonAnnounceIP
 	dropReasonNotMatchInterface
 )
+
+// DefaultInterfacesToExclude the interfaces in the list not be used to service announce.
+var DefaultInterfacesToExclude = []string{
+	"docker.*", "cbr.*", "dummy.*",
```

Comment:
I am a bit torn. On one side, this should help rather than not, on the other I am a bit afraid we are going to hit issues. Can we expose this as overrideable with a configmap?

Reply:
The idea for these changes came from calico, which automatically ignores these commonly used virtual NICs. see https://github.com/projectcalico/calico/blob/82f2f8ba06a07dc2c5cfc87de4e0397a82b0f92d/node/pkg/lifecycle/startup/autodetection/autodetection_linux.go#L18

Is there anything here that might cause us to hit issue?

Reply:
nope, I am just paranoid :-)